### ISSUE TITLE
A11y : Cannot easily drag/reorder rows in Value Mappings modal via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46494]
**Steps to reproduce**:

try to reorder the rows inside the value mappings modal using only a keyboard. 

**Actual Result**:

It is possible but when the drag handle is active only the left button is in focus. 

**Expected Result**

workflow accessible for keyboard-only users

Might be more intuitive if the whole row was focused?

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:
